app/grants/templates/grants/landing/landing_qf_active.html

```diff
@@ -22,7 +22,7 @@ <h4 class="text-wrap gc-font-base font-weight-normal mb-md-5 mb-4 pt-md-4 pt-1 t
       {% if show_main_round_banner %}
         <img src="{% static 'v2/images/grants/gr14-banner.png' %}" class="d-block w-100"/>
       {% else %}
-        <img src="{% static 'v2/images/grants/rocket-right.svg' %}" height="241" width="311" />
+        <img src="{% static 'v2/images/grants/garden_qf_landing.png' %}" height="auto" width="100%" />
```

Comment:
garden_qf_landing.png is currently a 5.5Mb image.

Might be good to try and get this down to < 250Kb

Reply:
Found a tool that seems to preserve the transparency:  https://resizing.app/features/resize-png/

Below is one I've resized to width=600px:

![garden_qf_landing](https://user-images.githubusercontent.com/19786448/186416410-84168f4b-b723-46f6-9d4f-eaa5ae56c14c.png)